docs/markdown/Reference-tables.md

```diff
@@ -104,6 +106,7 @@ set in the cross file.
 | arm                 | 32 bit ARM processor     |
 | avr                 | Atmel AVR processor      |
 | c2000               | 32 bit C2000 processor   |
+| dsp                 | 32 bit TI DSP processor  |
```

Comment:
`dsp` seems a pretty generic term to me... But on another hand, I'm not very familiar with this. I just wonder if the cpu name should be more specific, like `tidsp`?